fastlane/Fastfile

```diff
@@ -102,6 +102,11 @@ platform :android do
 
     UI.verbose("Deploying Custom Entitlements Computation version")
 
+    # Remove amazon module and dependents from project to publish custom-entitlement-computation library
+    sh("sed -i '' -e '/feature:amazon/d' settings.gradle")
+    sh("sed -i '' -e '/examples:purchase-tester/d' settings.gradle")
+    sh("sed -i '' -e '/api-tester/d' settings.gradle")
```

Comment:
This is honestly not great... But it should work for now... Ideally we would figure out a way to not build the publishing tasks in the amazon module, but I didn't find a way to do this. And changing the `ANDROID_VARIANT_TO_PUBLISH` after the tasks have been created shouldn't work either since it seems that is used at configuration time...

This should work for now until we figure out a better way to do this.